lib/src/utils/crypto-utils.ts

```diff
@@ -0,0 +1,147 @@
+import {
+    CryptoUtils,
+    DecodedIDTokenPayload,
+    JWKInterface,
+    SUPPORTED_SIGNATURE_ALGORITHMS
+} from "@asgardeo/auth-js";
+import base64url from "base64url";
+import sha256 from "fast-sha256";
+import * as jose from "jose";
+import randombytes from "secure-random-bytes";
+import { AsgardeoAuthException } from "../exception";
+
+export class NodeCryptoUtils implements CryptoUtils {
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    public constructor() { }
+
+    /**
+     * Get URL encoded string.
+     *
+     * @param {CryptoJS.WordArray} value.
+     * @returns {string} base 64 url encoded value.
+     */
+    public base64URLEncode(value: Buffer | string): string {
```

Comment:
We may have to modify this to incorporate the update to the CryptoUtils class. 
Refer:
1. https://github.com/asgardeo/asgardeo-auth-spa-sdk/blob/ea7cc845a42f5ef982c9b84d18f18749f6c81e26/lib/src/utils/crypto-utils.ts
2. https://github.com/asgardeo/asgardeo-auth-js-sdk#cryptoutils
3. https://github.com/asgardeo/asgardeo-auth-js-sdk/blob/master/lib/src/models/crypto.ts